### PR TITLE
Updated language worker protobuf version to 1.5.6

### LINKED ISSUE #824: Update protobuf to latest

### PR DESCRIPTION
**Pull request by andystaples**
### Issue describing the changes in this PR

resolves #824 

### Pull request checklist

* [x] My changes **do not** require documentation changes
  * [ ] Otherwise: Documentation issue linked to PR
* [x] My changes **should not** be added to the release notes for the next release
  * [ ] Otherwise: I've added my notes to `release_notes.md`
* [ ] My changes **do not** need to be backported to a previous version
  * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [x] I have added all required tests (Unit tests, E2E tests)

### Additional information

Additional PR information

### CODE REVIEW

File: protobuf/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# https://git-scm.com/docs/gitignore#_pattern_format
+
+
+
+# AZURE FUNCTIONS TEAM
+# For all file changes, github would automatically
+# include the following people in the PRs.
+# Language owners should get notified of any new changes to the proto file.
+
+src/proto/FunctionRpc.proto @vrdmr @gavin-aguiar @YunchuWang @surgupta-msft @satvu @ejizba @alrod @anatolib @kaibocai @shreyas-gopalakrishna @amamounelsayed @Francisco-Gamino
```
Comment (Francisco-Gamino) [inline protobuf/CODEOWNERS:14]: I am not sure if this file should be included with this code change. @ejizba , do you know?

Comment (ejizba) [reply protobuf/CODEOWNERS:14]: Should be fine - the process for updating protobuf pulls in all files for the repo. I assume you followed these steps, right Andy? https://github.com/Azure/azure-functions-language-worker-protobuf#pulling-updates

Comment (andystaples) [reply protobuf/CODEOWNERS:14]: Yup

Comment (Francisco-Gamino) [reply protobuf/CODEOWNERS:14]: Thank you @ejizba for confirming.

**Review by Francisco-Gamino — APPROVED** [review]
LGTM

**Pull request merged**